third_party/license.txt

```diff
@@ -2535,14 +2535,14 @@ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
---------------------------------------------------------------------------------
-
-
-
---------------------------------------------------------------------------------
-vendor github.com argoproj argo LICENSE (vendor/github.com/argoproj/argo/LICENSE)
---------------------------------------------------------------------------------
+
+--------------------------------------------------------------------------------
+
+
+
+--------------------------------------------------------------------------------
+vendor github.com argoproj argo LICENSE (vendor/github.com/argoproj/argo-workflows/LICENSE)
```

Comment:
This seems to be the only real change in this file, while others changes are just whitespace changes? Can we revert others changes?

Reply:
Thanks for pointing this out. As it turns out, this license.txt file uses Windows line endings and VSCode updates line endings to use the OS default. This has been corrected.